Tests/BodegaTests/DiskStorageTests.swift

```diff
@@ -112,6 +112,67 @@ final class DiskStorageTests: XCTestCase {
         XCTAssert(allKeys[3].value == "3")
         XCTAssert(allKeys.count == 10)
     }
+    
+    func testModificationDate() async throws {
+        // Make sure the modificationDate is nil if the key hasn't been stored
+        var mod = await storage.modificationDate(key: Self.testCacheKey)
```

Comment:
This is a personal nit but I like to use variables that are a little less terse, so instead of `mod` I'd prefer 
we use `modificationDate` so it's a little more obvious when scanning.

Reply:
Done